kim_python_utils/ase/core.py

```diff
@@ -1056,6 +1061,7 @@ def _query_aflow_designation_from_label_and_species(self):
             else:
                 self.library_prototype_label.append(None)
             if "short-name.source-value" in parameter_set:
+                # TODO: Will all parameter sets contain identical short names? Also should we extend instead of append below?
```

Comment:
No, parameter sets may have different short-names, and some may not have short-names (library-prototype-label and short-name have a 1-to-1 correspondence with each other, and correspond to curated sets of parameter-values in the AFLOW database). So everything below is correct -- it should append None if there is no short-name, and append a single-element list if there is one, resulting in a list of lists and Nones. The only issue is I have a typo and I'm setting short-name to library-prototype-label! I will fix it.